pcsx2/windows/VCprojects/pcsx2.vcxproj.filters

```diff
@@ -877,6 +877,9 @@
     <ClCompile Include="..\..\ps2\PGPU.cpp">
       <Filter>System\Ps2</Filter>
     </ClCompile>
+    <ClCompile Include="..\..\Mdec.cpp">
```

Comment:
The 2 MDEC files (cpp and h) were already in cmakelist but commented out. I can put them back in.